epochStart/bootstrap/epochStartMetaBlockProcessor.go

```diff
@@ -180,27 +239,90 @@ func (e *epochStartMetaBlockProcessor) GetEpochStartMetaBlock(ctx context.Contex
 		}
 	}()
 
-	err = e.requestMetaBlock()
+	err = e.waitForMetaBlock(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	err = e.waitForConfMetaBlock(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	chanRequests := time.After(durationBetweenReRequests)
 	chanCheckMaps := time.After(durationBetweenChecks)
+
```

Comment:
from the original process loop now we have three, but not sure we need 3 process loops.
Can you check if we can simplify?

With minimum changes from the previous version I would see the following flow:
- previously there was a request for the epoch start metablock and decide (statistically) which is the correct one. We were using the same process loop for both the request and validation that we have a "consensus" on the epoch start block.

- now in addition to the epoch start meta block, we also need its confirmation block (to take the proof for the previous block). To do this we don't need to decide statistically if there is a consensus on that block as well, it just needs to fit on top of the decided epoch start meta block.

Reply:
updated to remove separated flow for confirmation block